CHANGES.rst

```diff
@@ -6,6 +6,12 @@ There's a frood who really knows where his towel is.
 2.2.1 (unreleased)
 ^^^^^^^^^^^^^^^^^^
 
+- The solution was to change the selector "$ le" to "le" because it did not take the charged element, only those created (fixes `#871 <https://github.com/collective/collective.cover/issues/871>`_).
```

Comment:
these changelog entries are wrong; please fix them.

Reply:
I don't know if the recent changes in "Travis" affect the tests, I corrected the error, adding these variables to the environment:
    export ZSERVER_PORT = 55001
   export LISTENER_HOST = 49999

With this, the robot tests run me without any problem, in fact I already did some simple tests of mine